.clang-format

```diff
@@ -0,0 +1,145 @@
+---
+Language: Cpp
+# BasedOnStyle:  LLVM
+AccessModifierOffset: -2
+AlignAfterOpenBracket: Align
+AlignArrayOfStructures: None
+AlignConsecutiveAssignments: Consecutive
```

Comment:
Should be `None`, I think.

```bash
$ git grep -n '  =' -- :^winlibs :^maclibs :^MSBuild *.hh *.cc | wc -l
187
``` 

And these matches are mostly limited to a few source files: articleview.cc, mouseover.cc, sapi.hh, uiauto.hh.

Reply:
fixed